src/main/kotlin/uk/gov/justice/digital/hmpps/personintegrationapi/common/config/UserEnhancedOAuth2ClientCredentialGrantRequestConverter.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.personintegrationapi.config
+
+import org.springframework.http.HttpMethod
+import org.springframework.http.RequestEntity
+import org.springframework.security.oauth2.client.endpoint.OAuth2ClientCredentialsGrantRequest
+import org.springframework.security.oauth2.client.endpoint.OAuth2ClientCredentialsGrantRequestEntityConverter
+import org.springframework.util.MultiValueMap
+import java.util.*
```

Comment:
We have a bit of a digital studio standard of avoiding wildcard imports.  There's an IntelliJ setting Editor -> Code Style -> Kotlin... and then make sure `Use single name import` is selected everywhere.

Reply:
I had wildcard imports turned off, but I'd totally missed the override section further down which used wildcard for all java.utls...